### PR TITLE
ZCS-1115 Junit4 updates

### PR DESCRIPTION
**Pull request by tmclane**

### CODE REVIEW

File: store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java [hunk 1]
```diff
@@ -72,10 +84,14 @@
     private int mImapSslPort;
     private Map<Socket, BufferedReader> mReaders = new HashMap<Socket, BufferedReader>();
 
-    @Override
+    @Before
     public void setUp() throws Exception {
         SocketFactories.registerProtocols(true);
         mProv = Provisioning.getInstance();
+        if (!TestUtil.accountExists(USER_NAME))
+        {
```
Comment (grishick) [inline store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java]: formatting: "{" should be on the line above

File: store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java [hunk 2]
```diff
@@ -23,23 +23,35 @@
...
 
 import com.zimbra.cs.account.Provisioning;
 import com.zimbra.cs.account.Server;
 
 import static com.zimbra.common.net.SocketFactories.dummySSLSocketFactory;
 
-public final class TestPop3ImapAuth extends TestCase {
+public final class TestPop3ImapAuth  {
+
+    private static final String USER_NAME = "user1";
```
Comment (grishick) [inline store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java]: please use test-specific user names

File: store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java [hunk 3]
```diff
@@ -23,23 +23,36 @@
...
 import com.zimbra.cs.account.Server;
 
 import static com.zimbra.common.net.SocketFactories.dummySSLSocketFactory;
 
-public final class TestPop3ImapAuth extends TestCase {
+public final class TestPop3ImapAuth  {
+    @Rule
+    public static TestName testInfo = new TestName();
 
+    private static final String USER_NAME = String.format("%s-%s-user1", TestPop3ImapAuth.class.getSimpleName(), testInfo.getMethodName()).toLowerCase();
```
Comment (tmclane) [inline store/src/java/com/zimbra/qa/unittest/TestPop3ImapAuth.java]: Ok I'll move the line into setUp.

**Comment by tmclane** [comment]
@gordyt @grishick @ilyusha This has been updated. Please re-review.

**Pull request merged**